cocos/ui/scroll-view.ts

```diff
@@ -1457,15 +1458,15 @@ export class ScrollView extends ViewGroup {
         const viewTrans = this.view;
         const uiTrans = this._content._uiProps.uiTransformComp!;
         if (this._verticalScrollBar && this._verticalScrollBar.isValid) {
-            if (uiTrans.height < viewTrans.height) {
+            if (uiTrans.height <= viewTrans.height) {
```

Comment:
Comparison of floating point numbers requires the use of approximate comparisons.